Buildings/Fluid/Interfaces/TwoPortHeatMassExchanger.mo

```diff
@@ -96,7 +96,7 @@ equation
       points={{1,0},{100,0}},
       color={0,127,255}));
   connect(port_a, preDro.port_a) annotation (Line(
-      points={{-100,0},{-90,0},{-90,0},{-80,0},{-80,0},{-60,0}},
+      points={{-100,0},{-90,0},{-80,0},{-60,0}},
```

Comment:
Please revert this change as it may otherwise give conflict with other branches.